compose-shadow/src/androidMain/kotlin/com/adamglin/composeshadow/utils/NativePaint.android.kt

```diff
@@ -1,11 +1,17 @@
 package com.adamglin.composeshadow.utils
 
 import android.graphics.BlurMaskFilter
+import android.graphics.PorterDuffXfermode
 import android.os.Build
 import androidx.annotation.RequiresApi
 import androidx.compose.ui.graphics.NativePaint
+import org.jetbrains.skia.BlendMode
```

Comment:
Our thoughts are aligned, the only difference is that I believe skiko should not be introduced in common.

I think `androidx.compose.ui.graphics.BlendMode` should be used here instead, and the skiko dependency declared in commonMain should be removed.

Reply:
commonMain only uses `androidx.compose.ui.graphics.BlendMode`

In androidMain, convert `androidx.compose.ui.graphics.BlendMode` to `PorterDuff.Mode`

In other platforms, convert `androidx.compose.ui.graphics.BlendMode` to `skia.BlendMode.`

The benefit of this is that it does not introduce unnecessary skiko dependencies in androidMain.